### ISSUE TITLE
Error with vendor:publish

### ISSUE THREAD

**Issue by sylouuu** [getsentry/sentry-laravel#11]
Hey,

```
php artisan vendor:publish --provider="Sentry\SentryLaravel\SentryLaravelServiceProvider"
PHP Fatal error:  Uncaught exception 'ErrorException' with message 'array_merge(): Argument #2 is not an array' in /Users/syl/Sites/pro/aotd-website/vendor/sentry/sentry-laravel/src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php:72
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'array_merge(): ...', '/Users/syl/Site...', 72, Array)
#1 /Users/syl/Sites/pro/aotd-website/vendor/sentry/sentry-laravel/src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php(72): array_merge(Array, NULL)
#2 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Sentry\SentryLaravel\SentryLaravelServiceProvider->Sentry\SentryLaravel\{closure}(Object(Illuminate\Foundation\Application), Array)
#3 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#4 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundat in /Users/syl/Sites/pro/aotd-website/vendor/sentry/sentry-laravel/src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php on line 72
PHP Fatal error:  Uncaught exception 'ErrorException' with message 'array_merge(): Argument #2 is not an array' in /Users/syl/Sites/pro/aotd-website/vendor/sentry/sentry-laravel/src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php:72
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'array_merge(): ...', '/Users/syl/Site...', 72, Array)
#1 /Users/syl/Sites/pro/aotd-website/vendor/sentry/sentry-laravel/src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php(72): array_merge(Array, NULL)
#2 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Sentry\SentryLaravel\SentryLaravelServiceProvider->Sentry\SentryLaravel\{closure}(Object(Illuminate\Foundation\Application), Array)
#3 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#4 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundat in /Users/syl/Sites/pro/aotd-website/vendor/sentry/sentry-laravel/src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php on line 72
```

Using Laravel 5.2.

Got this in laravel.log:

```
[2016-05-06 09:40:05] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Sentry\SentryLaravel\SentryLaravelServiceProvider' not found' in /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(133): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(118): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php(114): Illuminate\Foundation\ProviderRepository->createProvider()
#4 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php(60): Illuminate\Foundation\ProviderRepository->compileManifest()
#5 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Foundation\ProviderRepository->load()
#6 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#7 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(203): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap()
#8 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(208): Illuminate\Foundation\Application->bootstrapWith()
#9 /Users/syl/Sites/pro/aotd-website/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(105): Illuminate\Foundation\Console\Kernel->bootstrap()
#10 /Users/syl/Sites/pro/aotd-website/artisan(36): Illuminate\Foundation\Console\Kernel->handle()
#11 /Users/syl/Sites/pro/aotd-website/artisan(0): {main}()
#12 {main}  
```

Bests

**Comment by Stayallive** [comment]
Fixed by #9 (when tagged)

**Comment by sylouuu** [comment]
No more error with 0.3.0.

Great work!
